src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -170,6 +276,37 @@ object ConductrPlugin extends AutoPlugin {
   def sandboxStop(): Unit =
     Process(Seq("sandbox", "stop")).!
 
+  /**
+   * A convenience function that waits on ConductR to become available.
+   * @param duration the max time to wait.
+   */
+  def waitForConductr(implicit duration: FiniteDuration): Unit = {
+    println("Waiting for ConductR to start...")
```

Comment:
It would be nice if we could provide users with feedback that something is happening in the background, like printing a dot every 500 milliseconds (we do exactly that when starting/stopping stuff in the lagom plugin, see https://github.com/lagom/lagom/blob/master/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/run/Servers.scala#L137)